omero/sysadmins/version-requirements.txt

```diff
@@ -1229,9 +1229,9 @@ Ice
       - from June 2015
       - to TBA
       - Unsupported
-      - Unsupported
+      - Supported
       - Recommended
-      - :zerocforum:`Ice 3.6.0 <announcements/6631-ice-3-6-0-and-ice-touch-3-6-0-released>`
```

Comment:
Could you add the link, rather than replace the previous link, as done for previous releases in the table?  I've found it quite useful to be able to find the old release on several occasions for earlier ice versions.